packages/fscomponents/src/components/ChangePasswordComponentFK.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { StyleProp, TextInputProps, View, ViewStyle } from 'react-native';
+import { Button } from './Button';
+import { Formik, FormikProps } from 'formik';
+import * as yup from 'yup';
+import { style as S } from '../styles/FormFK';
+import { TextField } from './FormikFields/TextField';
+import FSI18n, { translationKeys } from '@brandingbrand/fsi18n';
+const componentTranslationKeys = translationKeys.flagship.changePassword;
+
+
+export interface ChangePasswordPropsFK {
+  fieldsStyleConfig?: any; // the custom stylesheet we want to merge with the default stylesheet
+  onSubmit?: (value: any) => void; // the behaviour we want onpress of submit button
+  submitButtonStyle?: any;
+  submitTextStyle?: any;
+  submitText?: any; // Text to override the submit button
+  style?: any;
+  fieldsOptions?: any; // any extra desired behaviour, like placeholders
+  value?: any;
+  fieldStyle?: any;
+  labelStyle?: any;
+  containerStyle?: StyleProp<ViewStyle>;
+}
+
+export type FieldOption = {
+  [key in keyof FormValues]: TextInputProps;
+};
+
+export interface FormValues {
+  currentPassword: string;
+  confirmPassword: string;
+  newPassword: string;
+}
+
+
+const formSchema = yup.object().shape({
+  currentPassword: yup
+    .string()
+    .required()
+    .min(6)
+    .label('Current Password'),
+  confirmPassword: yup
+    .string()
+    .required()
+    .label('Confirm Password')
+    .test('password-match', 'Password must match ya fool', function(value: string): boolean {
```

Comment:
This error message seems a bit aggressive, especially because it doesn't look like you can change it.